### PR TITLE
change wizard, list and draggable repeater to use renderer

### PR DESCRIPTION
**Pull request by Blackbaud-LuisBello**

### CODE REVIEW

File: src/modules/wizard/wizard.component.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
     }
   }
 
-  sky-contrib-wizard-header, sky-contrib-wizard-header h1 {
+  .sky-contrib-wizard-header, .sky-contrib-wizard-header h1 {
```
Comment (blackbaud-joshgerdes) [inline src/modules/wizard/wizard.component.scss]: two lines

File: src/modules/list-repeater/list-repeater-item.component.ts [hunk 2]
```diff
@@ -12,7 +19,20 @@ export class SkyListRepeaterItemComponent {
   @Input() isExpandable: boolean = false;
   @Input() showContent: boolean = false;
   @Input() id: string;
+  @Input() item: any;
   isExpanded: boolean = true;
```
Comment (blackbaud-joshgerdes) [inline src/modules/list-repeater/list-repeater-item.component.ts]: put above inputs

File: src/modules/draggable-repeater/draggable-repeater-item.component.ts [hunk 3]
```diff
@@ -7,10 +16,23 @@ import { Component, Input, HostBinding } from '@angular/core';
...
+  @Input() itemContentTemplate: TemplateRef<any>;
+  @ContentChildren(forwardRef(() => SkyContribDraggableRepeaterItemContentComponent)) itemContentNode:
+    QueryList<SkyContribDraggableRepeaterItemContentComponent>;
+  @ContentChildren(forwardRef(() => SkyContribDraggableRepeaterItemTitleComponent)) itemTitleNode:
+    QueryList<SkyContribDraggableRepeaterItemTitleComponent>;
+  @ContentChildren(forwardRef(() => SkyContribDraggableRepeaterItemLeftComponent)) itemLeftNode:
+    QueryList<SkyContribDraggableRepeaterItemLeftComponent>;
+  @ContentChildren(forwardRef(() => SkyContribDraggableRepeaterItemRightComponent)) itemRightNode:
+    QueryList<SkyContribDraggableRepeaterItemRightComponent>;
   isExpanded: boolean = true;
```
Comment (blackbaud-joshgerdes) [inline src/modules/draggable-repeater/draggable-repeater-item.component.ts]: put above inputs

**Comment by Blackbaud-LuisBello** [comment]
updated

**Pull request merged**